README.md

```diff
@@ -414,6 +414,34 @@ temporary credentials were issued in. To account for the region specificity of
 the bucket, you may want to include the region name at then end of the bucket
 name. That way you can have consistent bucket names across regions.
 
+Create an IAM role to be used by API Gateway in order to enable CloudWatch logs.
+This needs to be done once per region in which CloudWatch logs are enabled.
+
+1. Navigate to the *IAM Management Console*
+
+2. Click *Roles* under *Access management*
+
+3. Click *Create role*
+
+4. Click *AWS service* for *Select type of trusted entity*
+
+5. Click *API Gateway* for *Choose a use case*, click *Next: Permissions*
+
+6. Ensure *AmazonAPIGatewayPushToCloudWatchLogs* is listed exclusively as the
+policy and click *Next: Tags*
+
+7. Add *name: azul-api_gateway* as a tag and click *Next: Review*
+
+8. For *Role name* enter `azul-api_gateway`, ensure *Policies* are as prescribed
+in step 6 and click *Create role*
+
+9. Be sure to copy the *Role ARN* for the newly created `azul-api_gateway` role
+and go to the *API Gateway* console
+
+10. Select any available Gateway resource e.g. `azul-{lambda}-{stage}`, click on
+*Settings* at the bottom of the left menu, paste the copied *Role ARN* into the
+*CloudWatch log role ARN* and click *Save*
```

Comment:
I'm confused about this step. You're supposed to set the ARN for *any* available Gateway resource? You do it for one and it activates the logging role for everything? Can you explain how this works?

Reply:
note to self: Add `to modify region wide settings do…` or something like it.